### PR TITLE
🌍  install and integrate leaflet map

### PR DESCRIPTION
**Pull request by alicepetzinger**
- Install and Integrate my Leaflet for react 
- connect with my storybook. 🌍

css->tbc

i commandos:
npm i react-leaflet leafle
npm i -D @types/leaflet

Sources:
[Leaflet for React - Start Setup](https://react-leaflet.js.org/docs/start-setup/)
[Leaflet for React - Start Installation](https://react-leaflet.js.org/docs/start-installation/ )

image

### CODE REVIEW

File: src/stories/pages/Map/Map.module.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.map {
+  height: 720px;
+  width: 390;
```
Comment (AnkePfei2fer) [inline src/stories/pages/Map/Map.module.css:3]: does it work? I think you should add a unit here, like px

**Review by AnkePfei2fer — COMMENTED** [review]
nice, that you got your leaflet map running! 🤖

File: src/stories/pages/Map/Map.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      className={styles.map}
+      center={[51.505, -0.09]}
+      zoom={13}
+      scrollWheelZoom={false}
+    >
+      <TileLayer url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png" />
+      <Marker position={[51.505, -0.09]}>
+        <Popup>
+          Visit Rosenheim <br />
+        </Popup>
```
Comment (marwinburesch) [inline src/stories/pages/Map/Map.tsx:18]: Something doesn't seem to work. Maybe the Marker image?
image

File: src/stories/pages/Map/Map.module.css [hunk 3]
```diff
@@ -0,0 +1,5 @@
+.map {
+  height: 720px;
+  width: 390px;
```
Comment (marwinburesch) [inline src/stories/pages/Map/Map.module.css:3]: These should be relative units

**Review by marwinburesch — COMMENTED** [review]
Nice work!

File: src/stories/pages/Map/Map.module.css (hunk 3 above)
Comment (alicepetzinger) [reply src/stories/pages/Map/Map.module.css:3]: Thank u, I changed it to em. 👍

File: src/stories/pages/Map/Map.tsx (hunk 2 above)
Comment (alicepetzinger) [reply src/stories/pages/Map/Map.tsx:18]: Hi, that's a integrated popup where i put 'Visit Rosenheim' inside for testing. I'll check it out 👍

**Pull request merged**